app/routes/statement.js

```diff
@@ -1,31 +1,34 @@
-const { getFileStream } = require('../storage')
-const joi = require('joi')
-const boom = require('@hapi/boom')
+const Boom = require('@hapi/boom')
+
+const schema = require('./schemas/statement')
+
+const getReadThroughStatement = require('../statement')
 
 module.exports = {
   method: 'GET',
   path: '/statement/{version}/{filename}',
+  options: {
+    validate: {
+      params: schema,
+      failAction: async (request, h, error) => {
+        return Boom.notFound(error)

```

Comment:
Should this be a bad request error as the schema validates the request has the required params and the correct format.  It's not checking if the resource exists at this point